### PR TITLE
Update to Laravel 5.1.

### PR DESCRIPTION
**Pull request by DFurnes**
#### Changes
Updates Aurora to Laravel 5.1, so it's like our other apps.

For review: @angaither 

img_0309-edit

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+DB_USERNAME=homestead
+DB_PASSWORD=secret
+
+CACHE_DRIVER=file
+SESSION_DRIVER=file
+QUEUE_DRIVER=sync
+
+NORTHSTAR_URL=http://northstar.dev:8000
+NORTHSTAR_APP_ID=456
+NORTHSTAR_API_KEY=abc4324
```
Comment (weerd) [inline .env.example]: should this be in here?

Comment (DFurnes) [inline .env.example]: It's the key we use in our [database seeder](https://github.com/DoSomething/northstar/blob/c9ea278ec3b8509a934be94595caacc7c7840318/database/seeds/ApiKeyTableSeeder.php#L17-L21), so I think it's fine.

Comment (angaither) [inline .env.example]: sure! they aren't the real ones!

Comment (weerd) [inline .env.example]: :+1:

**Comment by angaither** [comment]
what a day! :100:

**Pull request merged**